web/themes/contrib/civictheme/includes/menu.inc

```diff
@@ -62,6 +45,16 @@ function _civictheme_preprocess_block__navigation(array &$variables): void {
 
   $menu_name = strtr($variables['elements']['#derivative_plugin_id'], '-', '_');
 
+  $menu_theme_list = [
+    'civictheme_primary_navigation',
```

Comment:
@joshua-salsadigital Yes we got there!!! - this is working exactly how it should work.
There is a small improvement we need to do...
`civictheme_preprocess_block__menu_block__civictheme_secondary_navigation`

We add `ct-flex-justify-content-end` when in header - but is this basically whenever it is not `in_sidebar`? If so we should add another type of menu...
`civictheme_secondary_navigation` is for the header secondary nav, `civictheme_sidebar_navigation` is for the sidebar version and split out this hook into separate hooks for each type.